standings.js

```diff
@@ -61,13 +61,24 @@ fetchJSON("standings.json", function (gameResults) {
       return byWinningPercentage
     })
 
+const gamesBehindByTeam = sortedStandings.map((team) => {
```

Comment:
This variable isn't being used after it's declared (except for the `console.log` statement below, which doesn't really count), which is a sign that something is wrong.

Reply:
I think I can just chain the .map to the end of the previous variable, since the previous variable is sortedStandings and this one is just a map of sortedStandings.

Reply:
Never mind, it didn't like when I tried to do that, because sortedStandings is referenced in that map.